src/labels/CustomLabels.labels

```diff
@@ -1531,8 +1531,8 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <categories>Recurring-Donations, Error</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Validate RD Contact update when a closed opportunity exists on the RD</shortDescription>
-        <value>You can't change the Contact on a Recurring Donation that has Closed Opportunities.</value>
+        <shortDescription>Validate RD Contact/Account update when a closed opportunity exists on the RD</shortDescription>
+        <value>You can't change the donor information on a Recurring Donation that has Closed Opportunities.</value>
```

Comment:
@sam-knox Based on the discussion, I updated the error message to make it less confused when the user only change the Account on the RD update. Please review the new error message.

Reply:
@andrewyu-salesforce - Please update the value to:

"You can't change the Account or Contact on a Recurring Donation that has Closed Opportunities."

Reply:
@andrewyu-salesforce - Please update this label to:

"You can't change the Household Account or Contact on a Recurring Donation that has Closed Opportunities."

Reply:
Updated

Reply:
@andrewyu-salesforce - Excellent, thanks!